stdlib/2and3/pdb.pyi

```diff
@@ -19,3 +20,41 @@ else:
 
 def post_mortem(t: Optional[Any] = ...) -> None: ...
 def pm() -> None: ...
+
+class Pdb(Cmd):
+    if sys.version_info >= (3, 6):
+        def __init__(
+            self,
+            completekey: str = ...,
+            stdin: Optional[IO[str]] = ...,
+            stdout: Optional[IO[str]] = ...,
+            skip: Iterable[str] = ...,
+            nosigint: bool = ...,
+            readrc: bool = ...,
+        ) -> None: ...
+    elif sys.version_info >= (3, 2):
+        def __init__(
+            self,
+            completekey: str = ...,
+            stdin: Optional[IO[str]] = ...,
+            stdout: Optional[IO[str]] = ...,
+            skip: Iterable[str] = ...,
+            nosigint: bool = ...,
+        ) -> None: ...
+    else:
+        def __init__(
+            self,
+            completekey: str = ...,
+            stdin: Optional[IO[str]] = ...,
+            stdout: Optional[IO[str]] = ...,
+            skip: Iterable[str] = ...,
+        ) -> None: ...
+    def run(self, statement: str, globals: Optional[Dict[str, Any]] = ...,
+            locals: Optional[Dict[str, Any]] = ...) -> None: ...
+    def runeval(self, expression: str, globals: Optional[Dict[str, Any]] = ...,
+                locals: Optional[Dict[str, Any]] = ...) -> Any: ...
+    def runcall(self, *args: Any, **kwds: Any) -> Any: ...
+    if sys.version_info >= (3, 7):
+        def set_trace(self, *, header: Optional[str] = ...) -> None: ...
+    else:
+        def set_trace(self) -> None: ...
```

Comment:
At least in CPython HEAD, the run methods and `set_trace()` are actually defined on bdb.Bdb, from which Pdb derives. Since we don't have a stub for that yet, could you add a fixme or todo comment to that effect above?

Reply:
It seems I didn't save my other comment. The documentation is wrong here. `Bdb.set_trace()` is defined differently from the global `pdb.set_trace()`. It requires a `frame` argument instead and has no `header` argument: https://github.com/python/cpython/blob/f40b4a0b6277b2779b9ded3736325489f2af93e4/Lib/bdb.py#L321